package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "root",
+  "private": true,
+  "devDependencies": {
+    "lerna": "^3.22.1"
+  }
+}
```

Comment:
same with this

Reply:
I suppose the same issue happened with each one of this files, I somehow added them all to be committed. Which is a bit weird because `git add .` is supposed to commit only the modified, new or untracked files.